### PR TITLE
Tag label component

### PR DESCRIPTION
**Pull request by blackinno**
-reverse tag-label component

### CODE REVIEW

**Comment by blackinno** [comment]
Update tag label components by spec

File: packages/TagLabel/TagLabel.vue
Comment (antho1404) [inline packages/TagLabel/TagLabel.vue]: we should avoid `div` for labels, they are supposed to be inlined.
just `<span><slot/></span>` should be enough

File: packages/TagLabel/TagLabel.vue
Comment (antho1404) [inline packages/TagLabel/TagLabel.vue]: this computed is not needed.
let's just use the `type` property as a class

File: packages/TagLabel/TagLabel.vue
Comment (antho1404) [inline packages/TagLabel/TagLabel.vue]:
```suggestion
    <span class="tag" :class="type">
```

**Review by antho1404 — DISMISSED** [review]
the default design is not managed

File: packages/TagLabel/TagLabel.vue
Comment (blackinno) [reply packages/TagLabel/TagLabel.vue]: Fixed

File: packages/TagLabel/TagLabel.vue
Comment (blackinno) [reply packages/TagLabel/TagLabel.vue]: Fixed

File: packages/TagLabel/TagLabel.vue
Comment (blackinno) [reply packages/TagLabel/TagLabel.vue]: Fixed

File: storybook/stories/TagLabel.stories.js
Comment (antho1404) [inline storybook/stories/TagLabel.stories.js:8]: please put just some texts in the tag, it doesn't make sense to have h1-2-3

File: packages/TagLabel/TagLabel.vue
Comment (antho1404) [inline packages/TagLabel/TagLabel.vue]: Tag is full width which is not good, it's should be just the size of the text. Probably remove the display flex and align-items and/or add a display inline block

File: storybook/stories/TagLabel.stories.js
Comment (blackinno) [reply storybook/stories/TagLabel.stories.js:8]: Fixed

File: packages/TagLabel/TagLabel.vue
Comment (blackinno) [reply packages/TagLabel/TagLabel.vue]: Fixed

**Comment by EmmanuelDrouin** [comment]
- Updated font-size
- Updated padding tag
- Updated height tag
- Clean tag stories

@antho1404 ready for me, waiting for your approval

File: packages/TagLabel/TagLabel.vue
Comment (antho1404) [inline packages/TagLabel/TagLabel.vue]: This shouldn't exists. There are no `i` in the component so that's not the right place to put it.

File: packages/TagLabel/TagLabel.vue
Comment (antho1404) [inline packages/TagLabel/TagLabel.vue]: do not use harcoded values, this should be in the variables with a clear name

**Review by antho1404 — CHANGES_REQUESTED** [review]

**Comment by EmmanuelDrouin** [comment]
@antho1404 ready to check

**Review by antho1404 — APPROVED** [review]

**Pull request merged**
